pages/serverless-functions/reference-content/functions-sandbox.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Functions sandbox
+  description: Learn more about the sandboxing environments of Scaleway Serverless Functions.
+content:
+  h1: Functions sandbox
+  paragraph: Learn more about the sandboxing environments of Scaleway Serverless Functions.
+tags: functions sandbox sandboxing gvisor isolation mechanism serverless v1 v2 clock drift skew
+dates:
+  validation: 2025-03-13
+  posted: 2021-10-12
+categories:
+  - serverless
+  - functions
+---
+
+In the context of Scaleway Serverless Functions, a [Sandbox environment](/serverless-functions/concepts/#sandbox) is a critical security mechanism that isolates each function from others, ensuring that they operate in a secure and controlled space. Functions run in dedicated execution environments that prevent unauthorized access and potential interference between different resources.
+
+Scaleway Serverless Functions offers two sandbox environment options:
+
+- Sandbox v1 (legacy)
+- Sandbox v2 (recommended)
+
+## Sandbox v1
+
+Sandbox v1 is Serverless Functions' legacy sandboxing environment with slower [cold starts](/serverless-functions/concepts/#cold-start), but fully supports Linux system call interface. This option is recommended when processing large amounts of *syscalls*.
```

Comment:
```suggestion
Sandbox v1 is Serverless Functions' legacy sandboxing environment with slower [cold starts](/serverless-functions/concepts/#cold-start), but fully supports Linux system call interface.
```

Because when deploying something it's nearly impossible to determine which and how many syscalls you do